lib/sidekiq/statistic/locales/pt-br.yml

```diff
@@ -20,7 +20,7 @@ pt-br: # <---- change this to your locale code
   AverageSec: Média(s)
   MinTimeSec: Tempo Min(s)
   MaxTimeSec: Tempo Max(s)
-  LastJobStatus: Status do último trabalho
+  LastJobStatus: Status da última execução
```

Comment:
I believe the word "Execution" makes more sense in the Portuguese context once the "Job" word is closely related to "Work"